CairoMakie/src/utils.jl

```diff
@@ -239,7 +239,7 @@ function per_face_colors(
             wh = wsize .- 1
             cvec = map(uv) do uv
                 x, y = clamp.(round.(Int, Tuple(uv) .* wh) .+ 1, 1, wh)
-                return color[end - (y - 1), x]
```

Comment:
This was the way it is for textured meshes to work. Before/after:
![before](https://user-images.githubusercontent.com/10947937/167901650-6240bc68-1810-4b90-82a8-b287447e74db.png)
![after](https://user-images.githubusercontent.com/10947937/167901668-f45507f7-0284-4008-b533-842e4b50aec5.png)

Reply:
Oh good catch... We disabled all tests that check for this.. I was already wondering how this got changed without tests failing...

Reply:
```julia
using GLMakie, FileIO
cat = FileIO.load(Makie.assetpath("cat.obj"))
texture = FileIO.load(assetpath("diffusemap.png"))
mesh(cat, color = texture)

using CairoMakie
current_figure()
```